app/assets/stylesheets/application.scss

```diff
@@ -129,3 +129,9 @@ input[data-toggle]:checked + label:active {
 .hidden {
   display: none;
 }
+
+
+//Shareable bar
+.shareable {
+  background-color: #FDF3E7;
```

Comment:
Color literals like `#FDF3E7` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FDF3E7` should be written as `#fdf3e7`